tmpl/wiki/breaking-changes.md

```diff
@@ -1,5 +1,46 @@
 This page records API changes that require existing code to be updated.
 
+### 2015-09, Functoria edition
+
+#### Command line
+
+The options `--unix` and `--xen` are not available anymore.
+They must be replaced respectively by `-t unix` and `-t xen`.
+This option was available before so this is retro-compatible.
+
+The config file must be passed with the `-f` option (instead of being just
+an argument).
+
+#### Deprecation: Misc functions
+
+`get_mode` is deprecated. You should use keys instead. And in particular `Key.target` and `Key.is_xen`.
+
+`add_to_ocamlfind_libraries` and `add_to_opam_packages` are deprecated. Both the `foreign` and the `register` functions now possess the `~libraries` and `~packages` arguments to specify libraries dependencies.
+
+#### Entropy
+
+If you were using `tls` without the conduit combinator, you will be
+greeted during configuration by a message like this:
+
+```
+The "tls" library is loaded but entropy is not enabled!
+Please enable the entropy by adding a dependency to the nocrypto device.
+You can do so with the ~dependencies argument of Mirage.foreign.
+```
```

Comment:
That message has [been updated](https://github.com/mirage/mirage/blob/master/lib/mirage.ml#L753-L756) to:

```
  "The \"nocrypto\" library is loaded but entropy is not enabled!@ \
       Please enable the entropy by adding a dependency \
       to the nocrypto device. You can do so by adding [~deps:nocrypto] \
       to the arguments of Mirage.foreign."
```